fetchPeeps.js

```diff
@@ -0,0 +1,18 @@
+const fetchPeeps = () => {
+  fetch('https://chitter-backend-api-v2.herokuapp.com/peeps')
+  .then(response => response.json())
+  .then(data => {
+    data.forEach((peep) => {
+      newPeep = document.createElement("div");
+      newContent = document.createTextNode(`On ${peep['created_at']}, ${peep['user']['handle']} posted: ${peep['body']}`);
+      newPeep.className = 'post';
+      idNumber = document.querySelectorAll('.post').length + 1
+      newPeep.id = 'post-' + idNumber;
```

Comment:
Only possible issue is that you'd want the id to remain constant once set, if a user deleted a peep, would this not impact this?